public/index.html

```diff
@@ -0,0 +1,493 @@
+        <span class="drawn-arrow-wh">&nbsp;</span>
+      </div>
+      <div class="review-gallery sixth-skew">
+        <div>
+          <p>Eleifend morbi orci velit porttitor sed imperdiet ac ullamcorper Fusce eget tortor vel magna iaculis convallis. Aeneaneros vitae metus pellentesque tincidunt.Cum sociis natoque penatibus et magnis dis parturient montes turpis donec tincidunt tellus.</p>
+          <figure class="profile">
+            <span class="circle">
+              <img src="images/edison.jpg" alt="Portrait of Thomas Edison">
+            </span>
+            <figcaption><span class="name">Thomas Edison</span>Inventor : The Lightbulb</figcaption>
+          </figure>
+        </div>
+        <div>
+          <p>Eleifend morbi orci velit porttitor sed imperdiet ac ullamcorper Fusce eget tortor vel magna iaculis convallis. Aeneaneros vitae metus pellentesque tincidunt.Cum sociis natoque penatibus et magnis dis parturient montes turpis donec tincidunt tellus.</p>
+          <figure class="profile">
+            <span class="circle">
+              <img src="images/buchanan.jpg" alt="Portrait of James Buchanan">
+            </span>
+            <figcaption><span class="name">James Buchanan</span>Winner : Best Hair</figcaption>
+          </figure>
+        </div>
+        <div>
+          <p>Eleifend morbi orci velit porttitor sed imperdiet ac ullamcorper Fusce eget tortor vel magna iaculis convallis. Aeneaneros vitae metus pellentesque tincidunt.Cum sociis natoque penatibus et magnis dis parturient montes turpis donec tincidunt tellus.</p>
+          <figure class="profile">
+            <span class="circle">
+              <img src="images/stanton.jpg" alt="Portrait of Elizabeth Cady Stanton">
+            </span>
+            <figcaption><span class="name">Elizabeth Stanton</span>Champion : Women's Suffrage</figcaption>
+          </figure>
+        </div>
+      </div>
+    </section>
+		<section id="screens-nav" class="screenshots screenshot-section-wrapper eight-skew">
+      <h1 class="seven-skew">AppBox Screenshots<span class="accent-bar-C"></span></h1>
+      <div class="screenshot-gallery">
+        <div>
+          <img src="images/phone1.png" alt="Example screenshot of an app on a phone">
+        </div>
+        <div>
+          <img src="images/phone2.png" alt="Example screenshot of an app on a phone">
+        </div>
+        <div>
+          <img src="images/phone3.png" alt="Example screenshot of an app on a phone">
+        </div>
+        <div>
+          <img src="images/phone4.png" alt="Example screenshot of an app on a phone">
+        </div>
+      </div>
+    </section>
+    <section class="video-section nine-skew">
+      <h1>App Video Showcase<span class="accent-bar-C"></span></h1>
+      <video id="video1" src="video/mov_bbb.mp4">Your browser does not support HTML5 video.</video>
+    </section>
+    <section class="download-app-section ten-skew">
+			<div class="download-img">
```

Comment:
Tabbing went crazy here?!

Really, professionalism of your work will be judged by the care you give the spacing. Crazy, but true!